### ISSUE TITLE
Annotation tags problem

### ISSUE THREAD

**Issue by bensmrs** [ovh/ovh-warp10-datasource#36]
On Grafana 5.4.3, the annotation query is expected to return the tags in a list, not in a comma-separated string. This change is documented in v6.1, but was implemented before.
I wrote a fix here:
https://github.com/bensmrs/ovh-warp10-datasource/commit/93d23a354bc4774b3c0182e8c8466fac72c2f58e
but it shouldn't be backward compatible.
Would it be possible to check Grafana's version and format the tags accordingly?

**Comment by miton18** [comment]
Hi @bensmrs  !
There is a branch for Grafana versions, can you make a PR on the corresponding branch ?

**Comment by bensmrs** [comment]
Sure! Could you first create a 5.4 branch on your side? I don't think I can make a PR on a not yet existing branch…

**Comment by bensmrs** [comment]
Hi,
I solved another bug with Grafana 5+ on my fork, and it would be great if I could merge them on a new branch on this repo. So I hope you don't mind if I ask you again to create it (as I can't PR on a new branch)…

Also is it possible to have some hints on how to compile this project? I'm stuck with a `Duplicate identifier 'PropertyKey'` error so I had to manually edit both the .ts and the .js files.

Thanks!
